code/controllers/subsystem/disease.dm

```diff
@@ -15,27 +12,5 @@ SUBSYSTEM_DEF(disease)
 	if(!diseases)
 		diseases = subtypesof(/datum/disease)
 
-/datum/controller/subsystem/disease/Recover()
-	currentrun = SSdisease.currentrun
-	processing = SSdisease.processing
-	diseases = SSdisease.diseases
-	archive_diseases = SSdisease.archive_diseases
-
 /datum/controller/subsystem/disease/stat_entry(msg)
-	..("P:[processing.len]")
-
-/datum/controller/subsystem/disease/fire(resumed = 0)
-	if(!resumed)
-		src.currentrun = processing.Copy()
-	//cache for sanic speed (lists are references anyways)
-	var/list/currentrun = src.currentrun
-
-	while(currentrun.len)
-		var/datum/thing = currentrun[currentrun.len]
-		currentrun.len--
-		if(thing)
-			thing.process()
-		else
-			processing.Remove(thing)
-		if (MC_TICK_CHECK)
-			return
+	..("P:[active_diseases.len]")
```

Comment:
I kept this just for debugging purposes; useful to see how many diseases, in total, are active in all mobs..also for admins.